Source/ProgressionSystemRuntime/Private/Data/PSWorldSubsystem.cpp

```diff
@@ -127,25 +127,22 @@ void UPSWorldSubsystem::SetCurrentSpotComponent(UPSSpotComponent* MyHUDComponent
 		return;
 	}
 	PSCurrentSpotComponentInternal = MyHUDComponent;
-	UpdateProgressionActorsForSpot();
 }
 
 // Called when progression module ready
 void UPSWorldSubsystem::OnInitialized_Implementation()
 {
-	// Subscribe events on player type changed and Character spawned
-	BIND_ON_LOCAL_CHARACTER_READY(this, ThisClass::OnCharacterReady);
-
-	// Listen to handle input for each game state
-	BIND_ON_GAME_STATE_CHANGED(this, ThisClass::OnGameStateChanged);
-
 	StarDynamicProgressMaterial = UMaterialInstanceDynamic::Create(UPSDataAsset::Get().GetDynamicProgressionMaterial(), this);
 	if (!ensureMsgf(StarDynamicProgressMaterial, TEXT("ASSERT: [%i] %hs:\n'StarDynamicProgressMaterial' is null!"), __LINE__, __FUNCTION__))
 	{
 		return;
 	}
 
-	UpdateProgressionActorsForSpot();
+	// Subscribe events on player type changed and Character spawned
+	BIND_ON_LOCAL_CHARACTER_READY(this, ThisClass::OnCharacterReady);
```

Comment:
In this review, in `PSSpotComponent.cpp` above, you renamed `OnCharacterReady` to `OnLocalCharacterReady`, which I like as it makes function name straightforward. However, in `PSWorldSubsystem,cpp` you still use `OnCharacterReady`. Could you please rename it here as well to keep naming consistent?

Reply:
done https://github.com/h4rdmol/ProgressionSystem/pull/40/commits/d2b2758df734461bd3201696a6e924d768e44fd0